index.html

```diff
@@ -1492,6 +1492,8 @@ <h4>
       <a>NDEFRecord</a> interface:
     </p>
     <pre class="idl">
+      typedef (DOMString or BufferSource or NDEFMessageInit) NDEFRecordSource;
```

Comment:
I still feel `NDEFRecordDataSource` is more precise here.

As we already have `NDEFMessageSource` being used to provide all information needed to create `NDEFMessage`, so people may have the feeling that `NDEFRecordSource` contains all information for creating `NDEFRecord` accordingly, but in fact we use it to provide just the record payload data.

Reply:
@leonhsl Done.